src/lightly_train/_task_models/dinov2_eomt_semantic_segmentation/transforms.py

```diff
@@ -69,9 +69,10 @@ class DINOv2EoMTSemanticSegmentationTrainTransformArgs(
 
     image_size: tuple[int, int] = (518, 518)
     channel_drop: ChannelDropArgs | None = None
+    num_channels: int | Literal["auto"] = "auto"
     normalize: NormalizeArgs = Field(default_factory=NormalizeArgs)
     random_flip: RandomFlipArgs = Field(default_factory=RandomFlipArgs)
-    color_jitter: DINOv2EoMTSemanticSegmentationColorJitterArgs = Field(
+    color_jitter: DINOv2EoMTSemanticSegmentationColorJitterArgs | None = Field(
```

Comment:
Had to add `None` to be able to disable it because color jitter doesn't support more than 3 channels